design-system/atoms/input/adapters/default/default.variants.ts

```diff
@@ -0,0 +1,80 @@
+import { tv } from "tailwind-variants";
+
+export const InputDefaultVariants = tv({
+  slots: {
+    container: "flex w-full flex-col gap-sm",
+    wrapper: "flex w-full items-stretch justify-start",
+    base: [
+      "transition-all",
+      "flex w-full items-stretch justify-start gap-md rounded-md border-1 border-border-primary bg-background-primary-alt effect-box-shadow-xs",
+      "data-[hover=true]:border-border-primary-hover hover:border-border-primary-hover",
+    ],
+    contentWrapper: "flex flex-col items-start justify-center text-foreground-tertiary",
+    inputWrapper: "flex-1",
+    input: [
+      "h-full w-full bg-transparent outline-none focus:outline-none",
```

Comment:
```suggestion
      "h-full w-full bg-transparent outline-none outline-none",
```

To also handle `focus-visible`